src/components/figures/figure/figure.ts

```diff
@@ -160,7 +161,14 @@ export class FigureComponent extends Component<Props, SpreadsheetChildEnv> {
   }
 
   get wrapperStyle() {
-    const { x, y, width, height } = this.props.figureUI;
+    let { x, y, width, height, tag } = this.props.figureUI;
+    if (tag === "chart") {
```

Comment:
the figure component should be agnostic of the type of figure (chart, image, whatever).

The figure size  should be changed in the plugins when the user checks the "Show slicer" checkbox.
Another thing that doesn't work with the current approach: figure snapping doesn't take the slicer into account even though it should